src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -89,3 +90,63 @@
                                                 {:event  :wallet/get-suggested-routes
                                                  :error  error
                                                  :params request-params}))}]})))
+
+(rf/reg-event-fx :wallet/add-authorized-transaction
+ (fn [{:keys [db]} [transaction]]
+   (let [transaction-hashes (:hashes transaction)
+         chain-id           (key (first transaction-hashes))
+         tx-id              (first (val (first transaction-hashes)))
+         transaction-detes  {:status   :pending
+                             :id       (:id transaction)
+                             :chain-id chain-id}]
+     {:db (assoc-in db [:wallet :transactions tx-id] transaction-detes)
```

Comment:
Just a question here,

Are transactions associated to an account?
If so, please consider adding these transactions inside the respective account

Reply:
let's discuss this today!